lib/middleware.js

```diff
@@ -496,6 +497,18 @@ var middlewares = module.exports = {
     });
     res.end();
   },
+  imageSave: function (req, res) {
+    var stringStream = new StringStream('TARTARTARTARTAR');
+    res.setHeader('Content-Type', 'application/x-tar');
+    res.status(200);
+    stringStream.pipe(res);
+  },
+  imageLoad: function (req, res) {
+    req.on('data', noop);
+    req.on('end', function () {
+      res.sendStatus(200);
+    });
+  },
```

Comment:
updated to use real busy box tar (smallest image i could find, 2mb) now we also load this into the image array :)